File: exercicios/R/1079/1079.r
input = file('stdin','r')

n = as.integer(readLines(input,n=1))
i = 0
while(i < n){
    media = as.double(0)

    line = strsplit(readLines(input,n=1)," ")

    n1 = as.double(line[[1]][1])
    n2 = as.double(line[[1]][2])
    n3 = as.double(line[[1]][3])

    media = ((n1*2+n2*3+n3*5)/10)

    write(sprintf("%.1f",media),'')
    i = i + 1
}

File: exercicios/JavaScript/1079/1079.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var n = parseInt(lines.shift());

for(let i=0; i<n; i++){
    var media = 0;
    var line1 = lines.shift().split(' ');

    var a = parseFloat(line1[0]);
    var b = parseFloat(line1[1]);
    var c = parseFloat(line1[2]);

    media = ((a*2) + (b*3) + (c*5)) / 10;

    console.log(media.toFixed(1))
}